### PR TITLE
Jsx components conversion

### PR DESCRIPTION
**Pull request by Mohitk391**
Converted HTML,CSS to Jsx files for React Project.
Screenshots of the Screens are below
Screenshot (12)
Screenshot (13)
Screenshot (15)
Screenshot (17)

### CODE REVIEW

File: src/components/NavBar/NavBar.js [hunk 1]
```diff
@@ -7,13 +7,13 @@ function NavBar() {
...
                 <input className="navbar-searchbar" type="text" placeholder="Search for a product..."/>
-                <a href="#" ><i className="fa-solid fa-magnifying-glass"></i></a>
+                <a href="/#" ><i className="fa-solid fa-magnifying-glass"></i></a>
             </div>
             <div className="nav-bar-buttons">
                 <a href="/login.html"><button className="login">Login</button></a>
-                <a href="/wishlist.html" className="nav-bar-icon"><i className="fa fa-heart-o" aria-hidden="true"></i></a>
-                <a href="/cart.html" className="nav-bar-icon"><i className="fa fa-cart-plus" aria-hidden="true"></i></a>
-                <a  href="#" className="nav-bar-icon"><i className="fa fa-user" aria-hidden="true"></i></a>
+                <a href="/wishlist.html" className="nav-bar-icon"><i className="fa fa-heart-o" aria-hidden="true"></i> </a>
```
Comment (MayankKumar10) [inline src/components/NavBar/NavBar.js:14]: instead of anchor tag you can use link tag

File: src/pages/cart/Cart.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { NavBar } from "../../components/NavBar/NavBar";
 import "./cart.css";
+import {sofa} from "../../Assets/images/sofa.jpeg";
```
Comment (MayankKumar10) [inline src/pages/cart/Cart.js:3]: export all your image files to the index file and import image from it.

**Review by MayankKumar10 — APPROVED** [review]
Look into these Suggestion everything else Looks Good to me you can merge it

File: src/components/NavBar/NavBar.js (hunk 1 above)
Comment (Mohitk391) [reply src/components/NavBar/NavBar.js:14]: thank you for the suggestion, I did that in one of the next branches..

File: src/pages/cart/Cart.js (hunk 2 above)
Comment (Mohitk391) [reply src/pages/cart/Cart.js:3]: Okay will do that, thank you..

**Review by Ankur9669 — APPROVED** [review]
Screens are looking good, Great Work!

**Pull request merged**